bigbluebutton-html5/imports/startup/server/index.js

```diff
@@ -154,24 +165,19 @@ WebApp.connectHandlers.use('/feedback', (req, res) => {
 
     if (!user) {
       Logger.error(`Feedback failed, user with id=${userId} wasn't found`);
-      res.setHeader('Content-Type', 'application/json');
-      res.writeHead(500);
-      res.end(JSON.stringify({ status: 'ok' }));
-      return;
     }
 
+    res.setHeader('Content-Type', 'application/json');
+    res.writeHead(200);
+    res.end(JSON.stringify({ status: 'ok' }));
+
+    body.userName = user ? user.name : `[unconfirmed] ${reqUserName}`;
```

Comment:
wouldn't interesting add a boolean field `UserValidated` instead a name tag?

Reply:
What do you think about it @antobinary ?

Reply:
We do something similar here https://github.com/bigbluebutton/bigbluebutton/blob/2d66704bcf3d9e97df9790a83010cdbc24a45524/bigbluebutton-html5/imports/api/log-client/server/methods/logClient.js#L14-L23
But in this case I don't find it necessary to specify when the feedback was valid. I'd be OK to leave this part as is